docker/pytorch-aarch64/Dockerfile

```diff
@@ -290,13 +290,75 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 COPY scripts/build-pytorch.sh $PACKAGE_DIR/.
 COPY patches/onednn.patch $PACKAGE_DIR/.
 RUN $PACKAGE_DIR/build-pytorch.sh
+
+# Install torchvision, torchdata and torchtext
+RUN pip install --no-cache-dir torchvision==$TORCHVISION_VERSION \
+    torchdata==$TORCHDATA_VERSION
+ENV VENV_PACKAGE_DIR=$VENV_DIR/lib/python$PY_VERSION/site-packages
+COPY scripts/build-torchtext.sh $PACKAGE_DIR/.
+RUN $PACKAGE_DIR/build-torchtext.sh
+
+CMD ["bash", "-l"]
+# ========
+# Stage 4b: build PyTorch-xla
+# ========
+FROM pytorch-libs AS pytorch-xla
+ARG njobs
+ARG onednn_opt
+ARG default_py_version
+ARG cpu
+
+ENV ONEDNN_BUILD="${onednn_opt}" \
+    NP_MAKE="${njobs}" \
+    CPU="${cpu}"
+
+# Key version numbers
+ENV PY_VERSION="${default_py_version}" \
+    ONEDNN_VERSION="v2.6" \
+    TORCH_VERSION=1.11.0 \
+    TORCHVISION_VERSION=0.12.0 \
+    TORCHDATA_VERSION=0.3.0 \
+    TORCHTEXT_VERSION=0.12.0
+
+# Use a PACKAGE_DIR in userspace
+WORKDIR /home/$DOCKER_USER
+USER $DOCKER_USER
+ENV PACKAGE_DIR=/home/$DOCKER_USER/$PACKAGE_DIR
+RUN mkdir -p $PACKAGE_DIR
+
+# Copy in the Python virtual environment
+ENV VIRTUAL_ENV=/home/$DOCKER_USER/python3-venv
+COPY --chown=$DOCKER_USER:$DOCKER_USER --from=pytorch-tools $VIRTUAL_ENV /home/$DOCKER_USER/python3-venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# Get Bazel binary for AArch64 using the latest Bazelisk release
+RUN mkdir -p $PACKAGE_DIR/bazel && \
+   wget https://github.com/bazelbuild/bazel/releases/download/5.1.1/bazel-5.1.1-linux-arm64 -O $PACKAGE_DIR/bazel/bazel
+RUN chmod a+x $PACKAGE_DIR/bazel/bazel
+ENV export PATH="$PACKAGE_DIR/bazel:$PATH"
+
+# Build PyTorch and Torch-xla
+RUN pip install --no-cache-dir lark-parser
+COPY patches/0001-xla-cpu-Add-Arm-Compute-Library-ACL-build-option-for.patch $PACKAGE_DIR/.
```

Comment:
Yeah, I'm aware of the standard patching process with squashing into one. Here intentionally I left it, including the patch titles (didn't truncate) to clearly have 1-1 with the PR so that we know what we are building. I know this doesn't look neat. If you think squashing along with commit description is the preferred way, I will do that.